src/AstronautsTable.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import fetchJson from "./fetchJson";
+
+function AstronautsTable() {
+  const [error, setError] = useState(null);
+  const [astronauts, setAstronauts] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+      fetchJson()
+      .then(
+        (data) => {
+          setAstronauts(data);
+          setIsLoaded(true);
+        },
+        (error) => {
+        setError(error);
+        setIsLoaded(true)
+        }
+      );
+  }, []);
+
+  if (error) {
+    return <div>Error: {error.message}</div>;
+} else if (!isLoaded) {
+    return <div>Loading, please wait...</div>;
+} else {
+    console.log(astronauts.people);
+    const DisplayData=astronauts.people.map(
```

Comment:
Double-check whether this variable should be `camelCased` or `TitleCased`.

Also, it might be clearer to call this `astronautRow`.

Reply:
camelCase! And changed